packages/client/src/views/PrintCertificate/Payment.tsx

```diff
@@ -197,7 +198,7 @@ function mapStatetoProps(
   props: RouteComponentProps<{ registrationId: string; eventType: string }>
 ) {
   const { registrationId, eventType } = props.match.params
-  const event = getEvent(eventType)
+  const event = getEvent(eventType) as Event
```

Comment:
```suggestion
  const event = getEvent(eventType)
```
I noticed the return type of the function was Event